docs/data/deployments.yml

```diff
@@ -98,3 +98,18 @@
   dask: dask-kubernetes
   jupyter: Jupyterhub
   kind: cloud
+
+- name: GFDL Post-processing and Analysis Cluster
+  description: HPC cluster running at the NOAA Geophyiscal Fluid Dynamics Laboratory
+  platform: HPC
+  dask: dask-jobqueue with a custom PBSCluster subclass
```

Comment:
This interests me. If you have a moment, would you mind sharing a bit more about this. Either here, or preferably, on the dask-jobqueue issue tracker.

Reply:
Sure thing  - GFDL uses the [Moab Workload Manager](http://docs.adaptivecomputing.com/mwm/7-1-3/help.htm#cover_page.html) for batch job scheduling, which behaves very similarly to a standard PBS system (i.e. scripts take PBS directives etc.).  There are some differences, however, in the command names for submitting and cancelling jobs (see below).  As far as I can tell the only way to override those is by creating a subclass of PBSCluster.  In addition, I've found the need to override the function that parses the job ID, so that one can stop workers when desired (the default implementation does not parse it correctly).  The subclass is only a few lines:
```python
from dask_jobqueue import PBSCluster


class PPANCluster(PBSCluster):
    """A custom PBSCluster for use on the GFDL PPAN"""
    submit_command = 'msub'
    cancel_command = 'canceljob'

    def _job_id_from_submit_output(self, out):
        return out.strip()
```
I'm happy to report this in dask-jobqueue if you'd like.

Reply:
That's interesting!

That would be nice to report this in dask_jobqueue. Maybe others are using Moab.

Could you share this as a pull request? Maybe renaming the class as MoabCluster.

Reply:
I'd be open to that.  Before I do that, I'll just note that I think Moab is proprietary.  Does that pose any problems?

Reply:
No problems that I can see.

Reply:
Sounds good.  Here is a link to the PR: dask/dask-jobqueue#66.